[2025-09-16T01:00:04.501+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-16T01:00:04.665+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_bi.dbt_deps scheduled__2025-09-15T22:45:00+00:00 [queued]>
[2025-09-16T01:00:04.706+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_bi.dbt_deps scheduled__2025-09-15T22:45:00+00:00 [queued]>
[2025-09-16T01:00:04.709+0200] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-16T01:00:04.760+0200] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_deps> on 2025-09-15 22:45:00+00:00
[2025-09-16T01:00:04.784+0200] {standard_task_runner.py:64} INFO - Started process 277 to run task
[2025-09-16T01:00:04.791+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_api_to_bi', 'dbt_deps', 'scheduled__2025-09-15T22:45:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_bi.py', '--cfg-path', '/tmp/tmp0dzklo5m']
[2025-09-16T01:00:04.803+0200] {standard_task_runner.py:91} INFO - Job 10: Subtask dbt_deps
[2025-09-16T01:00:05.119+0200] {task_command.py:426} INFO - Running <TaskInstance: etl_api_to_bi.dbt_deps scheduled__2025-09-15T22:45:00+00:00 [running]> on host 1621268d5064
[2025-09-16T01:00:05.926+0200] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data' AIRFLOW_CTX_DAG_ID='etl_api_to_bi' AIRFLOW_CTX_TASK_ID='dbt_deps' AIRFLOW_CTX_EXECUTION_DATE='2025-09-15T22:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-15T22:45:00+00:00'
[2025-09-16T01:00:05.927+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-16T01:00:05.947+0200] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-16T01:00:05.951+0200] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "docker exec dbt bash -lc 'cd /usr/app && dbt deps --profiles-dir /root/.dbt'"]
[2025-09-16T01:00:05.976+0200] {subprocess.py:86} INFO - Output:
[2025-09-16T01:00:08.791+0200] {subprocess.py:93} INFO - [0m23:00:08  Running with dbt=1.8.3
[2025-09-16T01:00:10.125+0200] {subprocess.py:93} INFO - [0m23:00:10  Installing dbt-labs/dbt_utils
[2025-09-16T01:00:16.913+0200] {subprocess.py:93} INFO - [0m23:00:16  Installed from version 1.3.1
[2025-09-16T01:00:16.914+0200] {subprocess.py:93} INFO - [0m23:00:16  Up to date!
[2025-09-16T01:00:17.983+0200] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-16T01:00:17.985+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-16T01:00:18.039+0200] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_api_to_bi, task_id=dbt_deps, run_id=scheduled__2025-09-15T22:45:00+00:00, execution_date=20250915T224500, start_date=20250915T230004, end_date=20250915T230018
[2025-09-16T01:00:18.096+0200] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-16T01:00:18.138+0200] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-16T01:00:18.141+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
